packages/dita-example-sx-modules-spec-learning-xsd-learning2domain/src/api/insertNodeAndRemoveFromSiblingsCustomMutation.js

```diff
@@ -28,11 +29,11 @@ export default function insertNodeAndRemoveFromSiblings(argument, blueprint) {
 
 	blueprint.insertBefore(contextNode, newNode, referenceNode);
 
-	evaluateXPathToNodes('preceding-sibling::*', contextNode, blueprint)
-		.concat(evaluateXPathToNodes('following-sibling::*', contextNode, blueprint))
+	evaluateXPathToNodes(xq`preceding-sibling::*`, contextNode, blueprint)
+		.concat(evaluateXPathToNodes(xq`following-sibling::*`, contextNode, blueprint))
 		.forEach(function(siblingNode) {
 			var removeNode = evaluateXPathToFirstNode(
-				'child::' + argument.nodeName,
+				xq(`child::*[name() = ${argument.nodeName}]`),
```

Comment:
Missing quotes around ${argument.nodeName} Which is causing this to no longer work properly

This might not be the only place this went wrong. Search for "name() = ${", to find the other selectors.

Reply:
Agreed, I'm slowly coming back from that overload. It seemed nice at first, but it is annoying. It should indeed be `xq`child...``